docs/developers/keystore.md

```diff
@@ -0,0 +1,56 @@
+---
+id: keystore
+title: MultiversX Wallet - Keystore files
+---
+
+[comment]: # (mx-abstract)
+
+The keystore is a JSON file that encrypts a mnemonic (seed phrase). Until the user uses their password, the seed phrase remains encrypted, ensuring the safety and security of their transactions. The MultiversX keystore file provides users with a reliable and convenient method for managing their keys, protecting their assets, and ensuring the integrity of their transactions.
```

Comment:
_The MultiversX keystore is a JSON file that holds a mnemonic (seed phrase), encrypted with a password (as chosen by the user). Thus, the keystore provides users with a reliable and convenient method for managing their hot wallets and protecting their assets._

CC: @bogdan-rosianu

Reply:
changed